wagtailio/project_styleguide/templates/patterns/components/streamfields/quotes.html

```diff
@@ -0,0 +1,21 @@
+{% load wagtailcore_tags %}
+
+<div class="quotes">
+    <div class="quotes__header">
+        <p class="quotes__title heading-one">{{ value.title }}</p>
+        <div>
+            <svg aria-hidden="true" class="quotes__icon"><use xlink:href="#quotes"></use></svg>
+        </div>
+    </div>
+    <div class="quotes__grid">
+        {% for item in value.items.all %}
+        <div>
+            <p class="heading-four">{{ item.quote }}</p>
+            <span class="quotes__author">
+                {% if item.image %}<img class="quotes__author-image" src="{{ item.image.url }}">{% endif %}
+                <p class="mini-meta">{{ item.author|richtext }}</p>
```

Comment:
In my past experience variables with a `|richtext` filter need to have the parent element as a `div`, not a `p`, otherwise an empty `<p>` tag is output.

Should we add the `rich-text` class here as well?

Reply:
Ah rats, you're right. This was removed in https://docs.wagtail.org/en/stable/reference/contrib/legacy_richtext.html#module-wagtail.contrib.legacy.richtext